notebooks/02_foundations__connection_riemannian_metric.ipynb

```diff
@@ -0,0 +1,2266 @@
+{
```

Comment:
<p>NIT: If you moved --&gt; If you translate (importantly for later, we could also say: "if you parallel-transport..")</p><p>just because:</p><p>- there could be other ways of "moving" the vector, e.g. translating and rotating it</p><p>- it could be a good opportunity to start introducing the notion of parallel transport?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/geomstats/geomstats/pull/1551/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='14'/>

Reply:
<p>i agree</p>